DEBUG 2019-05-21 11:03:40,003: 	 Executing command: 'submit'
DEBUG 2019-05-21 11:03:40,004: 	 Could not find CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3; creating a new one.
DEBUG 2019-05-21 11:03:56,573: 	 Executing command: 'submit'
DEBUG 2019-05-21 11:03:56,573: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-21 11:03:56,582: 	 Checking credentials
DEBUG 2019-05-21 11:03:56,583: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-21 11:03:56,583: 	 Getting proxy life time left
DEBUG 2019-05-21 11:03:56,788: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 38867

 error: 
 retcode : 0
DEBUG 2019-05-21 11:03:56,892: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 38867

 error: 
 retcode : 0
DEBUG 2019-05-21 11:03:56,892: 	 Proxy valid for 10:47:47 hours
INFO 2019-05-21 11:03:56,892: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-21 11:03:56,893: 	 Loading CRAB configuration file.
ERROR 2019-05-21 11:03:56,938: 	 Error while loading CRAB configuration:
name 'os' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2019-05-21 11:03:56,938: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 87, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'os' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2019-05-21 11:04:12,608: 	 Executing command: 'submit'
DEBUG 2019-05-21 11:04:12,608: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-21 11:04:12,615: 	 Checking credentials
DEBUG 2019-05-21 11:04:12,616: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-21 11:04:12,616: 	 Getting proxy life time left
DEBUG 2019-05-21 11:04:12,720: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 38851

 error: 
 retcode : 0
DEBUG 2019-05-21 11:04:12,824: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 38851

 error: 
 retcode : 0
DEBUG 2019-05-21 11:04:12,825: 	 Proxy valid for 10:47:31 hours
INFO 2019-05-21 11:04:12,825: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-21 11:04:12,825: 	 Loading CRAB configuration file.
INFO 2019-05-21 11:04:14,734: 	 Wrong format in command-line argument '--dryrun'. Expected format is <section-name>.<parameter-name>=<parameter-value>.
INFO 2019-05-21 11:04:14,735: 	 If the argument '--dryrun' is an option to the submit command, try 'crab submit --dryrun [value for --dryrun option (if required)] [arguments]'.
ERROR 2019-05-21 11:04:14,735: 	 ERROR: Wrong command-line format.
ERROR 2019-05-21 11:04:14,735: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 65, in loadConfig
    raise ConfigurationException("ERROR: Wrong command-line format.")
ConfigurationException: ERROR: Wrong command-line format.
DEBUG 2019-05-21 11:04:33,206: 	 Executing command: 'submit'
DEBUG 2019-05-21 11:04:33,206: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-21 11:04:33,213: 	 Checking credentials
DEBUG 2019-05-21 11:04:33,213: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-21 11:04:33,213: 	 Getting proxy life time left
DEBUG 2019-05-21 11:04:33,317: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 38830

 error: 
 retcode : 0
DEBUG 2019-05-21 11:04:33,423: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 38830

 error: 
 retcode : 0
DEBUG 2019-05-21 11:04:33,423: 	 Proxy valid for 10:47:10 hours
INFO 2019-05-21 11:04:33,423: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-21 11:04:33,423: 	 Loading CRAB configuration file.
DEBUG 2019-05-21 11:04:34,684: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-21 11:04:34,684: 	 Cannot find the file crab_script.sh specified in the JobType.scriptExe configuration parameter.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2019-05-21 11:04:34,684: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 92, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Cannot find the file crab_script.sh specified in the JobType.scriptExe configuration parameter.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2019-05-21 13:36:11,702: 	 Executing command: 'submit'
DEBUG 2019-05-21 13:36:11,702: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-21 13:36:11,717: 	 Checking credentials
DEBUG 2019-05-21 13:36:11,718: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-21 13:36:11,718: 	 Getting proxy life time left
DEBUG 2019-05-21 13:36:11,922: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 29732

 error: 
 retcode : 0
DEBUG 2019-05-21 13:36:12,027: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 29732

 error: 
 retcode : 0
DEBUG 2019-05-21 13:36:12,027: 	 Proxy valid for 08:15:32 hours
INFO 2019-05-21 13:36:12,027: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-21 13:36:12,027: 	 Loading CRAB configuration file.
ERROR 2019-05-21 13:36:12,042: 	 Error while loading CRAB configuration:
name 'onfig' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2019-05-21 13:36:12,043: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 87, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'onfig' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2019-05-21 13:36:27,055: 	 Executing command: 'submit'
DEBUG 2019-05-21 13:36:27,055: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-21 13:36:27,089: 	 Checking credentials
DEBUG 2019-05-21 13:36:27,090: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-21 13:36:27,090: 	 Getting proxy life time left
DEBUG 2019-05-21 13:36:27,194: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 29716

 error: 
 retcode : 0
DEBUG 2019-05-21 13:36:27,298: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 29716

 error: 
 retcode : 0
DEBUG 2019-05-21 13:36:27,298: 	 Proxy valid for 08:15:16 hours
INFO 2019-05-21 13:36:27,299: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-21 13:36:27,299: 	 Loading CRAB configuration file.
ERROR 2019-05-21 13:36:27,719: 	 Error while loading CRAB configuration:
'Configuration' object has no attribute 'JobType'
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2019-05-21 13:36:27,720: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 87, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
'Configuration' object has no attribute 'JobType'
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2019-05-21 13:37:01,605: 	 Executing command: 'submit'
DEBUG 2019-05-21 13:37:01,605: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-21 13:37:01,612: 	 Checking credentials
DEBUG 2019-05-21 13:37:01,612: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-21 13:37:01,612: 	 Getting proxy life time left
DEBUG 2019-05-21 13:37:01,717: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 29682

 error: 
 retcode : 0
DEBUG 2019-05-21 13:37:01,822: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 29682

 error: 
 retcode : 0
DEBUG 2019-05-21 13:37:01,822: 	 Proxy valid for 08:14:42 hours
INFO 2019-05-21 13:37:01,823: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-21 13:37:01,823: 	 Loading CRAB configuration file.
DEBUG 2019-05-21 13:37:04,088: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-21 13:37:04,103: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_NanoPostprocessing_DYDileptonTest' already exists 
Please change the requestName in the config file
ERROR 2019-05-21 13:37:04,104: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_NanoPostprocessing_DYDileptonTest' already exists 
Please change the requestName in the config file
DEBUG 2019-05-21 13:38:41,270: 	 Executing command: 'submit'
DEBUG 2019-05-21 13:38:41,270: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-21 13:38:41,277: 	 Checking credentials
DEBUG 2019-05-21 13:38:41,277: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-21 13:38:41,277: 	 Getting proxy life time left
DEBUG 2019-05-21 13:38:41,380: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 29582

 error: 
 retcode : 0
DEBUG 2019-05-21 13:38:41,485: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 29582

 error: 
 retcode : 0
DEBUG 2019-05-21 13:38:41,485: 	 Proxy valid for 08:13:02 hours
INFO 2019-05-21 13:38:41,485: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-21 13:38:41,485: 	 Loading CRAB configuration file.
DEBUG 2019-05-21 13:38:42,865: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-21 13:38:42,865: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_NanoPostprocessing_DYDileptonTest' already exists 
Please change the requestName in the config file
ERROR 2019-05-21 13:38:42,866: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_NanoPostprocessing_DYDileptonTest' already exists 
Please change the requestName in the config file
DEBUG 2019-05-21 14:17:21,128: 	 Executing command: 'submit'
DEBUG 2019-05-21 14:17:21,128: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-21 14:17:21,135: 	 Checking credentials
DEBUG 2019-05-21 14:17:21,135: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-21 14:17:21,136: 	 Getting proxy life time left
DEBUG 2019-05-21 14:17:21,240: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 27262

 error: 
 retcode : 0
DEBUG 2019-05-21 14:17:21,344: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 27262

 error: 
 retcode : 0
DEBUG 2019-05-21 14:17:21,344: 	 Proxy valid for 07:34:22 hours
INFO 2019-05-21 14:17:21,344: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-21 14:17:21,344: 	 Loading CRAB configuration file.
ERROR 2019-05-21 14:17:21,738: 	 Keyboard Interrupted
DEBUG 2019-05-22 10:47:28,723: 	 Executing command: 'submit'
DEBUG 2019-05-22 10:47:28,724: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-22 10:47:28,731: 	 Checking credentials
DEBUG 2019-05-22 10:47:28,731: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-22 10:47:28,731: 	 Getting proxy life time left
DEBUG 2019-05-22 10:47:28,836: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 83929

 error: 
 retcode : 0
DEBUG 2019-05-22 10:47:28,940: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 83929

 error: 
 retcode : 0
DEBUG 2019-05-22 10:47:28,940: 	 Proxy valid for 23:18:49 hours
INFO 2019-05-22 10:47:28,941: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-22 10:47:28,941: 	 Loading CRAB configuration file.
ERROR 2019-05-22 10:47:28,954: 	 Error while loading CRAB configuration:
name 'campaign_name' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2019-05-22 10:47:28,954: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 87, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'campaign_name' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2019-05-22 10:49:03,981: 	 Executing command: 'submit'
DEBUG 2019-05-22 10:49:03,981: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-22 10:49:03,989: 	 Checking credentials
DEBUG 2019-05-22 10:49:03,989: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-22 10:49:03,989: 	 Getting proxy life time left
DEBUG 2019-05-22 10:49:04,093: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 83833

 error: 
 retcode : 0
DEBUG 2019-05-22 10:49:04,197: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 83833

 error: 
 retcode : 0
DEBUG 2019-05-22 10:49:04,198: 	 Proxy valid for 23:17:13 hours
INFO 2019-05-22 10:49:04,198: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-22 10:49:04,198: 	 Loading CRAB configuration file.
DEBUG 2019-05-22 10:49:05,696: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-22 10:49:05,697: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_NanoPostprocessing_DYDileptonTest' already exists 
Please change the requestName in the config file
ERROR 2019-05-22 10:49:05,697: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_NanoPostprocessing_DYDileptonTest' already exists 
Please change the requestName in the config file
DEBUG 2019-05-22 11:46:55,514: 	 Executing command: 'submit'
DEBUG 2019-05-22 11:46:55,514: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-22 11:46:55,544: 	 Checking credentials
DEBUG 2019-05-22 11:46:55,545: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-22 11:46:55,545: 	 Getting proxy life time left
DEBUG 2019-05-22 11:46:55,749: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 80362

 error: 
 retcode : 0
DEBUG 2019-05-22 11:46:55,853: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 80362

 error: 
 retcode : 0
DEBUG 2019-05-22 11:46:55,853: 	 Proxy valid for 22:19:22 hours
INFO 2019-05-22 11:46:55,853: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2019-05-22 11:46:55,853: 	 Loading CRAB configuration file.
DEBUG 2019-05-22 11:46:57,524: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-22 11:46:57,525: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_NanoPostprocessing_DYDileptonTest_DoubleMuon' already exists 
Please change the requestName in the config file
ERROR 2019-05-22 11:46:57,525: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_NanoPostprocessing_DYDileptonTest_DoubleMuon' already exists 
Please change the requestName in the config file
DEBUG 2019-05-22 15:10:51,826: 	 Executing command: 'status'
DEBUG 2019-05-22 15:10:51,826: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-28 13:57:57,628: 	 Executing command: 'kill'
DEBUG 2019-05-28 13:57:57,628: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
ERROR 2019-05-28 13:57:57,629: 	 Error: Please indicate the CRAB project directory with --dir=<project-directory>.
ERROR 2019-05-28 13:57:57,629: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 268, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 582, in validateOptions
    raise ex
MissingOptionException: [91mError[0m: Please indicate the CRAB project directory with --dir=<project-directory>.
DEBUG 2019-05-31 16:49:36,628: 	 Executing command: 'submit'
DEBUG 2019-05-31 16:49:36,629: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 16:49:36,637: 	 Checking credentials
DEBUG 2019-05-31 16:49:36,637: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 16:49:36,637: 	 Getting proxy life time left
DEBUG 2019-05-31 16:49:36,741: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 77903

 error: 
 retcode : 0
DEBUG 2019-05-31 16:49:36,845: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 77903

 error: 
 retcode : 0
DEBUG 2019-05-31 16:49:36,846: 	 Proxy valid for 21:38:23 hours
ERROR 2019-05-31 16:49:36,846: 	 CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/UWVV/Utilities/scripts/crab.py not found.
ERROR 2019-05-31 16:49:36,846: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 49, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/UWVV/Utilities/scripts/crab.py not found.
DEBUG 2019-05-31 16:50:23,501: 	 Executing command: 'submit'
DEBUG 2019-05-31 16:50:23,502: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 16:50:23,508: 	 Checking credentials
DEBUG 2019-05-31 16:50:23,509: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 16:50:23,509: 	 Getting proxy life time left
DEBUG 2019-05-31 16:50:23,613: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 77856

 error: 
 retcode : 0
DEBUG 2019-05-31 16:50:23,717: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 77856

 error: 
 retcode : 0
DEBUG 2019-05-31 16:50:23,717: 	 Proxy valid for 21:37:36 hours
ERROR 2019-05-31 16:50:23,717: 	 CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/UWVV/Utilities/scripts/crab.py not found.
ERROR 2019-05-31 16:50:23,718: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 49, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/UWVV/Utilities/scripts/crab.py not found.
DEBUG 2019-05-31 16:50:51,767: 	 Executing command: 'submit'
DEBUG 2019-05-31 16:50:51,767: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 16:50:51,774: 	 Checking credentials
DEBUG 2019-05-31 16:50:51,774: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 16:50:51,774: 	 Getting proxy life time left
DEBUG 2019-05-31 16:50:51,878: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 77828

 error: 
 retcode : 0
DEBUG 2019-05-31 16:50:51,983: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 77828

 error: 
 retcode : 0
DEBUG 2019-05-31 16:50:51,983: 	 Proxy valid for 21:37:08 hours
INFO 2019-05-31 16:50:51,983: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 16:50:51,983: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 16:51:57,710: 	 Executing command: 'submit'
DEBUG 2019-05-31 16:51:57,710: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 16:51:57,717: 	 Checking credentials
DEBUG 2019-05-31 16:51:57,718: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 16:51:57,718: 	 Getting proxy life time left
DEBUG 2019-05-31 16:51:57,822: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 77762

 error: 
 retcode : 0
DEBUG 2019-05-31 16:51:57,926: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 77762

 error: 
 retcode : 0
DEBUG 2019-05-31 16:51:57,926: 	 Proxy valid for 21:36:02 hours
INFO 2019-05-31 16:51:57,927: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 16:51:57,927: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 16:52:39,447: 	 Executing command: 'submit'
DEBUG 2019-05-31 16:52:39,447: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 16:52:39,454: 	 Checking credentials
DEBUG 2019-05-31 16:52:39,454: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 16:52:39,454: 	 Getting proxy life time left
DEBUG 2019-05-31 16:52:39,558: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 77720

 error: 
 retcode : 0
DEBUG 2019-05-31 16:52:39,663: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 77720

 error: 
 retcode : 0
DEBUG 2019-05-31 16:52:39,663: 	 Proxy valid for 21:35:20 hours
INFO 2019-05-31 16:52:39,663: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 16:52:39,663: 	 Loading CRAB configuration file.
ERROR 2019-05-31 16:52:39,887: 	 Error while loading CRAB configuration:
name 'haddscript' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2019-05-31 16:52:39,887: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 87, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'haddscript' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2019-05-31 16:56:57,345: 	 Executing command: 'submit'
DEBUG 2019-05-31 16:56:57,346: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 16:56:57,353: 	 Checking credentials
DEBUG 2019-05-31 16:56:57,353: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 16:56:57,353: 	 Getting proxy life time left
DEBUG 2019-05-31 16:56:57,457: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 77462

 error: 
 retcode : 0
DEBUG 2019-05-31 16:56:57,561: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 77462

 error: 
 retcode : 0
DEBUG 2019-05-31 16:56:57,561: 	 Proxy valid for 21:31:02 hours
INFO 2019-05-31 16:56:57,562: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 16:56:57,562: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 16:56:58,805: 	 Overriden parameter Data.inputDataset with '/DoubleMuon/Run2016B_ver2-Nano14Dec2018_ver2-v1/NANOAOD'
DEBUG 2019-05-31 16:56:58,805: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-31 16:56:58,806: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
ERROR 2019-05-31 16:56:58,806: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
DEBUG 2019-05-31 23:11:11,364: 	 Executing command: 'submit'
DEBUG 2019-05-31 23:11:11,364: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 23:11:11,371: 	 Checking credentials
DEBUG 2019-05-31 23:11:11,371: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 23:11:11,371: 	 Getting proxy life time left
DEBUG 2019-05-31 23:11:11,475: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 55008

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:11,579: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 55008

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:11,579: 	 Proxy valid for 15:16:48 hours
INFO 2019-05-31 23:11:11,579: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 23:11:11,580: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 23:11:13,237: 	 Overriden parameter Data.inputDataset with '/DoubleMuon/Run2016B_ver2-Nano14Dec2018_ver2-v1/NANOAOD'
DEBUG 2019-05-31 23:11:13,238: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-31 23:11:13,239: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
ERROR 2019-05-31 23:11:13,239: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
DEBUG 2019-05-31 23:11:13,496: 	 Executing command: 'submit'
DEBUG 2019-05-31 23:11:13,497: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 23:11:13,503: 	 Checking credentials
DEBUG 2019-05-31 23:11:13,503: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 23:11:13,503: 	 Getting proxy life time left
DEBUG 2019-05-31 23:11:13,607: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 55006

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:13,711: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 55006

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:13,711: 	 Proxy valid for 15:16:46 hours
INFO 2019-05-31 23:11:13,711: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 23:11:13,712: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 23:11:14,920: 	 Overriden parameter Data.inputDataset with '/DoubleMuon/Run2016C-Nano14Dec2018-v1/NANOAOD'
DEBUG 2019-05-31 23:11:14,920: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-31 23:11:14,921: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
ERROR 2019-05-31 23:11:14,921: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
DEBUG 2019-05-31 23:11:15,189: 	 Executing command: 'submit'
DEBUG 2019-05-31 23:11:15,189: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 23:11:15,195: 	 Checking credentials
DEBUG 2019-05-31 23:11:15,195: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 23:11:15,195: 	 Getting proxy life time left
DEBUG 2019-05-31 23:11:15,299: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 55004

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:15,403: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 55004

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:15,403: 	 Proxy valid for 15:16:44 hours
INFO 2019-05-31 23:11:15,403: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 23:11:15,403: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 23:11:16,606: 	 Overriden parameter Data.inputDataset with '/DoubleMuon/Run2016D-Nano14Dec2018-v1/NANOAOD'
DEBUG 2019-05-31 23:11:16,607: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-31 23:11:16,607: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
ERROR 2019-05-31 23:11:16,607: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
DEBUG 2019-05-31 23:11:16,864: 	 Executing command: 'submit'
DEBUG 2019-05-31 23:11:16,864: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 23:11:16,871: 	 Checking credentials
DEBUG 2019-05-31 23:11:16,871: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 23:11:16,871: 	 Getting proxy life time left
DEBUG 2019-05-31 23:11:16,974: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 55003

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:17,078: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 55002

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:17,078: 	 Proxy lifetime 55003 secs is different from voms extension lifetime 55002 secs for proxy: /tmp/x509up_u4301
DEBUG 2019-05-31 23:11:17,078: 	 Proxy valid for 15:16:42 hours
INFO 2019-05-31 23:11:17,079: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 23:11:17,079: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 23:11:18,348: 	 Overriden parameter Data.inputDataset with '/DoubleMuon/Run2016E-Nano14Dec2018-v1/NANOAOD'
DEBUG 2019-05-31 23:11:18,349: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-31 23:11:18,349: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
ERROR 2019-05-31 23:11:18,350: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
DEBUG 2019-05-31 23:11:18,591: 	 Executing command: 'submit'
DEBUG 2019-05-31 23:11:18,591: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 23:11:18,597: 	 Checking credentials
DEBUG 2019-05-31 23:11:18,597: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 23:11:18,598: 	 Getting proxy life time left
DEBUG 2019-05-31 23:11:18,702: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 55001

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:18,806: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 55001

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:18,806: 	 Proxy valid for 15:16:41 hours
INFO 2019-05-31 23:11:18,806: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 23:11:18,806: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 23:11:20,016: 	 Overriden parameter Data.inputDataset with '/DoubleMuon/Run2016F-Nano14Dec2018-v1/NANOAOD'
DEBUG 2019-05-31 23:11:20,017: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-31 23:11:20,017: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
ERROR 2019-05-31 23:11:20,017: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
DEBUG 2019-05-31 23:11:20,275: 	 Executing command: 'submit'
DEBUG 2019-05-31 23:11:20,275: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 23:11:20,282: 	 Checking credentials
DEBUG 2019-05-31 23:11:20,282: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 23:11:20,282: 	 Getting proxy life time left
DEBUG 2019-05-31 23:11:20,388: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 54999

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:20,492: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 54999

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:20,493: 	 Proxy valid for 15:16:39 hours
INFO 2019-05-31 23:11:20,493: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 23:11:20,493: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 23:11:21,693: 	 Overriden parameter Data.inputDataset with '/DoubleMuon/Run2016G-Nano14Dec2018-v1/NANOAOD'
DEBUG 2019-05-31 23:11:21,694: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-31 23:11:21,694: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
ERROR 2019-05-31 23:11:21,694: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
DEBUG 2019-05-31 23:11:21,939: 	 Executing command: 'submit'
DEBUG 2019-05-31 23:11:21,939: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
DEBUG 2019-05-31 23:11:21,946: 	 Checking credentials
DEBUG 2019-05-31 23:11:21,946: 	 Found proxy file /tmp/x509up_u4301
DEBUG 2019-05-31 23:11:21,946: 	 Getting proxy life time left
DEBUG 2019-05-31 23:11:22,050: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -timeleft
 output : 54998

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:22,153: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u4301 -actimeleft
 output : 54997

 error: 
 retcode : 0
DEBUG 2019-05-31 23:11:22,153: 	 Proxy lifetime 54998 secs is different from voms extension lifetime 54997 secs for proxy: /tmp/x509up_u4301
DEBUG 2019-05-31 23:11:22,153: 	 Proxy valid for 15:16:37 hours
INFO 2019-05-31 23:11:22,154: 	 Will use CRAB configuration file /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab.py
DEBUG 2019-05-31 23:11:22,154: 	 Loading CRAB configuration file.
DEBUG 2019-05-31 23:11:23,360: 	 Overriden parameter Data.inputDataset with '/DoubleMuon/Run2016H-Nano14Dec2018-v1/NANOAOD'
DEBUG 2019-05-31 23:11:23,361: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2019-05-31 23:11:23,361: 	 Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
ERROR 2019-05-31 23:11:23,361: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 315, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon' already exists 
Please change the requestName in the config file
DEBUG 2019-05-31 23:22:41,148: 	 Executing command: 'status'
DEBUG 2019-05-31 23:22:41,148: 	 Found CRAB cache file /afs/hep.wisc.edu/home/kdlong/.crab3
ERROR 2019-05-31 23:22:41,148: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon_Run2016C-Nano14Dec2018-v1
ERROR 2019-05-31 23:22:41,148: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 161, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 268, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 590, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1904/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 452, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/cms/kdlong/WZAnalysis/CMSSW_10_2_14/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/NanoVVSkims/crab/crab_UWVVNtuples_31May2019_DoubleMuon_Run2016C-Nano14Dec2018-v1
